src/component/feature/FeatureView2/FeatureView2.tsx

```diff
@@ -158,6 +170,24 @@ const FeatureView2 = () => {
                                 >
                                     <Archive />
                                 </PermissionIconButton>
+                                <PermissionIconButton
+                                    onClick={() => setOpenStaleDialog(true)}
+                                    permission={UPDATE_FEATURE}
+                                    projectId={projectId}
+                                    tooltip="Toggle stale status"
+                                    data-loading
+                                >
```

Comment:
Tooltip should be dynamic. Once you flip a toggle to stale, the tooltip should show `toggle to active` or smth similar.
<img width="1317" alt="Screen Shot 2021-11-10 at 9 43 49 PM" src="https://user-images.githubusercontent.com/47088512/141190428-da313d4a-2928-4ecd-839c-4bf1e744ad49.png">